src/Pages/Shop/ShopListCol.scss

```
@@ -0,0 +1,63 @@
+.ShopListCol {
+  display: flex;
+  width: 100vw;
+  padding: 20px;
+
+  aside {
+    width: 216px;
+    height: 100%;
+  }
+
+  button {
+    font-size: 10px;
+    padding: 0;
+
+    &:hover {
+      font-weight: 800;
+    }
+  }
+
+  main {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+
+    nav {
+      span,
+      a {
+        font-size: 10px;
+        color: black;
+        margin: 0 8px;
+      }
+      a:hover {
+        font-weight: 800;
+      }
+    }
+  }
+
+  header {
+    display: flex;
+    top: 0;
+    position: sticky;
+    justify-content: space-between;
+    height: 57px;
+    width: 100%;
+    font-size: 10px;
+    padding: 22px 5px;
+    background-color: white;
+  }
```

```suggestion
  header {
    display: flex;
    justify-content: space-between;
    position: sticky;
    top: 0;
    height: 57px;
    width: 100%;
    padding: 22px 5px;
    font-size: 10px;
    background-color: white;
  }
```
유사한 속성끼리 묶어주시고 중요한 속성을 위로 올려주세요.

---

속성정렬완료하였습니다